### PR TITLE
Closes #1839 dummy fg@main

### PR DESCRIPTION
**Pull request by Fanny-Gautier**
Thank you for your Pull Request! We have developed this task checklist from the [Development Process Guide](https://pharmaverse.github.io/admiral/CONTRIBUTING.html#detailed-development-process) to help with the final steps of the process. Completing the below tasks helps to ensure our reviewers can maximize their time on your code as well as making sure the admiral codebase remains robust and consistent.   

Please check off each taskbox as an acknowledgment that you completed the task or check off that it is not relevant to your Pull Request. This checklist is part of the Github Action workflows and the Pull Request will not be merged into the `main` branch until you have checked off each task.

- [x] Place Closes #<insert_issue_number> into the beginning of your Pull Request Title (Use Edit button in top-right if you need to update)
- [x] Code is formatted according to the [tidyverse style guide](https://style.tidyverse.org/). Run `styler::style_file()` to style R and Rmd files
- [x] Updated relevant unit tests or have written new unit tests, which should consider realistic data scenarios and edge cases, e.g. empty datasets, errors, boundary cases etc. - See [Unit Test Guide](https://pharmaverse.github.io/admiraldev/articles/unit_test_guidance.html#tests-should-be-robust-to-cover-realistic-data-scenarios)
- [x] If you removed/replaced any function and/or function parameters, did you fully follow the [deprecation guidance](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#deprecation)?
- [x] Review the [Cheat Sheet](https://github.com/pharmaverse/admiral/blob/main/inst/cheatsheet/admiral_cheatsheet.pdf). Make any required updates to it by editing the file `inst/cheatsheet/admiral_cheatsheet.pptx` and re-upload a PDF version of it to the same folder.
- [x] Update to all relevant roxygen headers and examples, including keywords and families. Refer to the [categorization of functions](https://pharmaverse.github.io/admiraldev/articles/programming_strategy.html#categorization-of-functions) to tag appropriate keyword/family.
- [x] Run `devtools::document()` so all `.Rd` files in the `man` folder and the `NAMESPACE` file in the project root are updated appropriately
- [x] Address any updates needed for vignettes and/or templates
- [x] Update `NEWS.md` under the header `# admiral (development version)` if the changes pertain to a user-facing function (i.e. it has an `@export` tag) or documentation aimed at users (rather than developers)
- [x] Build admiral site `pkgdown::build_site()` and check that all affected examples are displayed correctly and that all new functions occur on the "[Reference](https://pharmaverse.github.io/admiral/reference/index.html)" page. 
- [x] Address or fix all lintr warnings and errors - `lintr::lint_package()`
- [x] Run `R CMD check` locally and address all errors and warnings - `devtools::check()`
- [x] Link the issue in the Development Section on the right hand side.
- [x] Address all merge conflicts and resolve appropriately
- [x] Pat yourself on the back for a job well done! Much love to your accomplishment!

### CODE REVIEW

**Comment by Fanny-Gautier** [comment]
Build admiral site pkgdown::build_site() : ticked but failed with errors in RStudio

**Comment by bms63** [comment]
@Fanny-Gautier @Lina2689 

Click this button and see if that fixes the CI check for linter.  I thought we had updated that.

image


For pkg build issues.  I feel like this is an issue with your OS and RStudio.  Do you have anyone at Cytel you can talk to regarding  solutions?

File: R/my_first_fcn.R [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#' Derive on boarding dummy function
+#'
+#' @author   Fanny Gautier
```
Comment (bms63) [inline R/my_first_fcn.R:3]: No need to use author field

File: R/my_first_fcn.R [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+#'
+# Mandatory fields in header:
+#' @param    first_time TRUE or FALSE
+#' @details  outputs "Welcome to the admiral family!".
+#' @return   a Welcome message
+#' @export
+#' @keywords utils_print
+#' @family   utils_print
+#  keywords  der_message is not a known tag, so not included for this function
+#  family    der_message is not a known tag, so not included for this function
```
Comment (bms63) [inline R/my_first_fcn.R:13]: you can just remove these then

File: R/my_first_fcn.R [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+#' @examples
+#' library(testthat)
```
Comment (bms63) [inline R/my_first_fcn.R:15]: how come you are doing `library(testthat)`  ????

Oh I see you are using expect function ummm...I wonder if there is something else we could use here??

**Comment by bms63** [comment]
Apologies - the onboarding issue is a bit out of date.  We no longer need `@main`  at the end of a branch name.  

image

**Comment by Fanny-Gautier** [comment]
> @Fanny-Gautier @Lina2689
> 
> Click this button and see if that fixes the CI check for linter. I thought we had updated that.
> 
> image
> 
> For pkg build issues. I feel like this is an issue with your OS and RStudio. Do you have anyone at Cytel you can talk to regarding solutions?

This well fixes the CI issues. thanks.
image

**Comment by bms63** [comment]
Looks awesome @Fanny-Gautier!!  I think you should jump into a real issue now!!

**Comment by manciniedoardo** [comment]
Great job!

**Pull request closed without merging**